feature-libs/product/schematics/add-product/__snapshots__/index_spec.ts.snap

```diff
@@ -24,6 +24,44 @@ export class ProductBulkPricingFeatureModule { }
 "
 `;
 
+exports[`Spartacus Product schematics: ng-add BulkPricing feature general setup b2b features configuration should be added 1`] = `
+"import { NgModule } from '@angular/core';
+import { translationChunksConfig, translations } from \\"@spartacus/assets\\";
+import { FeaturesConfig, I18nConfig, OccConfig, provideConfig, SiteContextConfig } from \\"@spartacus/core\\";
+import { defaultB2bCheckoutConfig, defaultB2bOccConfig } from \\"@spartacus/setup\\";
+import { defaultCmsContentProviders, layoutConfig, mediaConfig } from \\"@spartacus/storefront\\";
+
+@NgModule({
+  declarations: [],
+  imports: [
+  ],
+  providers: [provideConfig(layoutConfig), provideConfig(mediaConfig), ...defaultCmsContentProviders, provideConfig(<OccConfig>{
+    backend: {
+      occ: {
+        baseUrl: 'https://localhost:9002',
+      }
+    },
+  }), provideConfig(<SiteContextConfig>{
+    context: {
+      currency: ['USD'],
+      language: ['en'],
+    },
+  }), provideConfig(<I18nConfig>{
+    i18n: {
+      resources: translations,
+      chunks: translationChunksConfig,
+      fallbackLang: 'en'
+    },
+  }), provideConfig(<FeaturesConfig>{
+    features: {
+      level: '4.1'
```

Comment:
why is the snapshot creating for 4.1 feature level

I see there's no default in the schema.json, so I'm assuming we usually pass a --featureLevel flag on snapshot creation?

Reply:
I don't know where but if there is a default somewhere, we should bump it up since there will be a lot of broken styling as it takes into account styling, features, templates, etc for <= 4.1

Reply:
It's because `"version": "4.1.0-next.0",` is set to the product's `package.json`.
This goes for all other features.